docs/build/tools/filecoin-backed-pinning.md

```diff
@@ -1,64 +1,64 @@
 ---
-title: Filecoin-backed IPFS Pinning Services
-description: A brief description of Filecoin-backed IPFS pinning services and how to use them.
+title: Filecoin-backed pinning
```

Comment:
Please undo the renaming in this section back to the original and rename the file as well!